okhttp-tests/src/test/java/okhttp3/internal/ws/RealWebSocketTest.java

```diff
@@ -79,6 +79,14 @@
     assertFalse(client.webSocket.send("Hello!"));
   }
 
+  @Test public void clientCloseWith0Fails() throws IOException {
+    try {
+      client.webSocket.close(0, null);
```

Comment:
This needs a `fail()` after it.

Reply:
Thanks, good catch :(

Reply:
https://github.com/square/okhttp/pull/3545